MapboxVision/src/test/java/com/mapbox/vision/dsl/VisionTestDsl.kt

```diff
@@ -9,27 +9,53 @@ annotation class VisionDslMarker
 object TestCase {
 
     private val givenBlocks = mutableListOf<DynamicContainer>()
+    private val givenBlockDisplayNames = mutableSetOf<String>()
+
     private val whenBlocks = mutableListOf<DynamicContainer>()
+    private val whenBlockDisplayNames = mutableSetOf<String>()
+
     private val thenBlocks = mutableListOf<DynamicTest>()
+    private val thenBlockDisplayNames = mutableSetOf<String>()
 
     operator fun invoke(block: TestContext.() -> Unit): List<DynamicContainer> {
         givenBlocks.clear()
+        givenBlockDisplayNames.clear()
         TestContext().run(block)
         return givenBlocks
     }
 
-    internal fun addGivenBlock(displayName: String) {
-        givenBlocks.add(DynamicContainer.dynamicContainer("Given: $displayName", whenBlocks.toMutableList()))
+    internal fun addGivenBlock(displayName: String, prefix: String = "Given") {
```

Comment:
why can we change `prefix`?  `addGivenBlock ` is obvious name for `Given`. And you could move `Given` to Consts or ENUM. The same for bottom functions 
```suggestion
    internal fun addGivenBlock(displayName: String) {
```

Reply:
fix it